<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enrolar/Borrar</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>291</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labe_Nom">
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Nom"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_ID">
        <property name="text">
         <string>PageID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ID">
        <property name="maxLength">
         <number>256</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxBase">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Base de Datos</string>
   </property>
   <widget class="QTextEdit" name="textEditBaseDatos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Enrolar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enrolar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
